db/schema.rb

```diff
@@ -807,6 +810,7 @@
     t.boolean  "require_ship_address",             default: true
     t.text     "description"
     t.string   "tracking_url",         limit: 255
+    t.integer  "tax_category_id"
```

Comment:
won't we use this column on queries from now on? if so, we might need to add an index covering it (compound or single).

Reply:
Looks like there's various other tables where we have a `tax_category_id` column and no index on it...

Definitely needs looking at :+1:

Reply:
I've added an index here for now but I notice a couple of things; we have various tables in the schema that are completely unused and we are probably missing lots of indexes on things that probably need them. Maybe we should make an issue to assess it.

Reply:
I tend to frame this sort of top-down; from the endpoint with the biggest cost and thus where perf. optimization is a priority and digging down to its queries, but perhaps hunting down the top 5 most critical missing indexes might take us 2h top and will likely have a big impact on the DB load :thinking: 

Are there any that you already identified and could become a first top-5 list?

Reply:
I couldn't help and I did a quick exploration. For the record, this is how I proceeded on a production server:

First I ran
```sql
select * from pg_stat_all_tables where schemaname = 'public' order by seq_scan desc limit 10;
```

and then I checked the statistics of the queries against the resulting tables like so:

```sql
select * from pg_stat_statements where query ilike '%spree_calculators%';
```

I expect to open a PR soon with the first results.